components/data_comps/datm/cime_config/config_component.xml

```diff
@@ -54,6 +54,7 @@
       <value compset="^RCP4_">rcp4.5</value>
       <value compset="^RCP2_">rcp2.6</value>
       <value compset="^HIST_">trans_1850-2000</value>
+      <value compset="^20TR_">trans_1850-2000</value>
```

Comment:
This is putting back a name (20TR) that we chose to replace with HIST.   I don't think we should do this - just use HIST.